src/Components/Artwork/FillwidthItem.tsx

```diff
@@ -115,7 +115,9 @@ export class FillwidthItemContainer extends React.Component<
   }
 }
 
-export const FillwidthItem = styled(FillwidthItemContainer)`
+export const FillwidthItem = styled(FillwidthItemContainer).attrs<
+  FillwidthItemContainerProps
+>({})`
```

Comment:
I see, just applying the props to the type?

Reply:
Yupp